src/libxrpl/protocol/STVar.cpp

```diff
@@ -205,6 +208,9 @@ STVar::STVar(SerializedTypeID id, SField const& name)
         case STI_UINT160:
             construct<STUInt160>(name);
             return;
+        case STI_UINT192:
+            construct<STUInt192>(name);
+            return;
```

Comment:
Another case of, "while you're making changes to this file, how about refactoring this duplicated code into a single function?"

Reply:
Wow. That code was not quite as duplicated as I thought it was. Still, looks like a good change. Thanks!